### PR TITLE
Transcription page UI cleanup

### PR DESCRIPTION
**Pull request by chloebrett**
Separates words into paragraphs; adds project name heading

Screenshot:

Screen Shot 2022-09-25 at 8 22 11 pm

### CODE REVIEW

**Comment by chloebrett** [comment]
Tests failing due to entirely unrelated ffmpeg issue

**Review by Irmy — CHANGES_REQUESTED** [review]
UI looks so good and the line breaks separate it so well! Can we please get the paragraph threshold to be increased to 4.0 seconds so that Zac can test it and give feedback. I at first suggested 2.0s but then he requested 4.0 or 5.0s.

File: src/sharedTypes.ts [hunk 1]
```diff
@@ -1,3 +1,11 @@
+// Transcription chunks are either a take group or a paragraph of words
+export type TranscriptionChunk = TakeGroup | Word[];
+
+export interface SelectionState {
+  self: IndexRange;
+  others: Record<ClientId, IndexRange>;
```
Comment (Mhudlark) [inline src/sharedTypes.ts:8]: The tests look like they're failing because `ClientId` is not defined. What is the ffmpeg issue?

Comment (chloebrett) [reply src/sharedTypes.ts:8]: I’ll take another look

**Comment by chloebrett** [comment]
@Irmy changed it to 4 seconds

**Comment by chloebrett** [comment]
FFMPEG error, only seems to be an issue locally not in CI:

```
 FAIL  src/main/export/__tests__/constructEDL.test.ts
  ● Test suite failed to run

    The name `ffmpeg-static` was looked up in the Haste module map. It cannot be resolved, because there exists several different files, or packages, that provide a module for that particular name and platform. The platform is generic (no extension). You must delete or exclude files until there remains only one of these:

      * `/Users/chloe/c/mlvet/release/build/mac-arm64/MLVET.app/Contents/Resources/ffmpeg-static/package.json` (package)
      * `/Users/chloe/c/mlvet/release/build/mac/MLVET.app/Contents/Resources/ffmpeg-static/package.json` (package)

    > 1 | import ffmpegStaticPath from 'ffmpeg-static';
        | ^
      2 | import ffprobeStatic from 'ffprobe-static';
      3 | import path from 'path';
      4 |

      at ModuleMap._assertNoDuplicates (node_modules/jest-haste-map/build/ModuleMap.js:230:11)
      at Object.<anonymous> (src/main/ffUtils.ts:1:1)
```

**Comment by chloebrett** [comment]
FFMPEG error, only seems to be an issue locally not in CI:

```
 FAIL  src/main/export/__tests__/constructEDL.test.ts
  ● Test suite failed to run

    The name `ffmpeg-static` was looked up in the Haste module map. It cannot be resolved, because there exists several different files, or packages, that provide a module for that particular name and platform. The platform is generic (no extension). You must delete or exclude files until there remains only one of these:

      * `/Users/chloe/c/mlvet/release/build/mac-arm64/MLVET.app/Contents/Resources/ffmpeg-static/package.json` (package)
      * `/Users/chloe/c/mlvet/release/build/mac/MLVET.app/Contents/Resources/ffmpeg-static/package.json` (package)

    > 1 | import ffmpegStaticPath from 'ffmpeg-static';
        | ^
      2 | import ffprobeStatic from 'ffprobe-static';
      3 | import path from 'path';
      4 |

      at ModuleMap._assertNoDuplicates (node_modules/jest-haste-map/build/ModuleMap.js:230:11)
      at Object.<anonymous> (src/main/ffUtils.ts:1:1)
```

**Comment by chloebrett** [comment]
Ready to land

**Review by Irmy — APPROVED** [review]

**Review by Mhudlark — APPROVED** [review]
Nice work!

**Pull request merged**